### PR TITLE
Update dependabot.yml

### PR DESCRIPTION
**Pull request by jennyliang220**
- Ignore minor upgrades for `cspell` for weekly check
- Add daily check for `specberus`
- Add `github-actions` weekly check

### CODE REVIEW

**Review by deniak — COMMENTED** [review]
is this going to ignore the patch releases for specberus? We want echidna to have the latest release of specberus including patch versions.
Otherwise, the PR looks good.

**Comment by jennyliang220** [comment]
> is this going to ignore the patch releases for specberus? We want echidna to have the latest release of specberus including patch versions. Otherwise, the PR looks good.

It was ignoring all the patch versions before the change.
```
- package-ecosystem: "npm"
  directory: "/"
  schedule:
    interval: "daily"
    time: '00:00'
  allow:
    - dependency-name: "specberus"
```
I believe this will check all the specberus versions upgrade daily, including patch changes.

**Review by deniak — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,3 +9,18 @@ updates:
...
+    interval: "daily"
+    time: '00:00'
+  allow:
+    - dependency-name: "specberus"
+- package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: weekly
+      time: '00:00'
+    open-pull-requests-limit: 10
```
Comment (jennyliang220) [inline .github/dependabot.yml:26]:
```suggestion
  directory: '/'
  schedule:
    interval: weekly
    time: '00:00'
  open-pull-requests-limit: 10
```

**Comment by jennyliang220** [comment]
> ```
> - package-ecosystem: "npm"
>   directory: "/"
>   schedule:
>     interval: "daily"
>     time: '00:00'
>   allow:
>     - dependency-name: "specberus"
> ```

This is causing error same as https://github.com/w3c/specberus/pull/1645. I'm removing specberus config for now, will add it when I found a way

File: .github/dependabot.yml [hunk 2]
```diff
@@ -9,3 +9,18 @@ updates:
...
+      - dependency-name: 'cspell'
+        update-types: ['version-update:semver-minor']
+- package-ecosystem: "npm"
+  directory: "/"
+  schedule:
+    interval: "daily"
+    time: '00:00'
+  allow:
+    - dependency-name: "specberus"
+- package-ecosystem: github-actions
```
Comment (jennyliang220) [inline .github/dependabot.yml:21]:
```suggestion
- package-ecosystem: github-actions
```

**Pull request merged**